<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Notepad</class>
 <widget class="QMainWindow" name="Notepad">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>685</width>
    <height>430</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Notepad</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QTextEdit" name="textEdit"/>
    </item>
   </layout>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <property name="sizePolicy">
    <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
     <horstretch>0</horstretch>
     <verstretch>0</verstretch>
    </sizepolicy>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSave_as"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionFont"/>
   <addaction name="actionBold"/>
   <addaction name="actionItalic"/>
   <addaction name="actionUnderline"/>
   <addaction name="actionEsquerda"/>
   <addaction name="actionDireita"/>
   <addaction name="actionCentro"/>
   <addaction name="actionCor"/>
   <addaction name="separator"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/new.png</normaloff>:/images/new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
   <property name="toolTip">
    <string>Novo Documento</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+N</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Abrir</string>
   </property>
   <property name="iconText">
    <string>Abrir</string>
   </property>
   <property name="toolTip">
    <string>Abrir arquivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+O</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Salvar</string>
   </property>
   <property name="toolTip">
    <string>Salvar arquivo</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="actionSave_as">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/save_as.png</normaloff>:/images/save_as.png</iconset>
   </property>
   <property name="text">
    <string>Salvar como</string>
   </property>
   <property name="toolTip">
    <string>Salvar arquivo como</string>
   </property>
   <property name="shortcut">
    <string>Alt+S</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset theme="exit.png" resource="notepad.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Sair</string>
   </property>
   <property name="toolTip">
    <string>Sair do notepad</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/edit_undo.png</normaloff>:/images/edit_undo.png</iconset>
   </property>
   <property name="text">
    <string>Desfazer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Z</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/edit_redo.png</normaloff>:/images/edit_redo.png</iconset>
   </property>
   <property name="text">
    <string>Refazer</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+Y</string>
   </property>
  </action>
  <action name="actionFont">
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/font.png</normaloff>:/images/font.png</iconset>
   </property>
   <property name="text">
    <string>Fonte</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+F</string>
   </property>
  </action>
  <action name="actionItalic">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/italic.png</normaloff>:/images/italic.png</iconset>
   </property>
   <property name="text">
    <string>Italico</string>
   </property>
   <property name="toolTip">
    <string>Italico</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+I</string>
   </property>
  </action>
  <action name="actionBold">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/bold.png</normaloff>:/images/bold.png</iconset>
   </property>
   <property name="text">
    <string>negrito</string>
   </property>
   <property name="toolTip">
    <string>Negrito</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+B</string>
   </property>
  </action>
  <action name="actionUnderline">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/underline.png</normaloff>:/images/underline.png</iconset>
   </property>
   <property name="text">
    <string>Sublinhado</string>
   </property>
   <property name="iconText">
    <string>Sublinhado</string>
   </property>
   <property name="toolTip">
    <string>Sublinhado</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="actionCor">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/cor.png</normaloff>:/images/cor.png</iconset>
   </property>
   <property name="text">
    <string>Cor</string>
   </property>
   <property name="toolTip">
    <string>Cor da fonte</string>
   </property>
  </action>
  <action name="actionEsquerda">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/left_align.png</normaloff>:/images/left_align.png</iconset>
   </property>
   <property name="text">
    <string>Esquerda</string>
   </property>
   <property name="toolTip">
    <string>Alinha a esquerda</string>
   </property>
  </action>
  <action name="actionDireita">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/right_align.png</normaloff>:/images/right_align.png</iconset>
   </property>
   <property name="text">
    <string>Direita</string>
   </property>
   <property name="toolTip">
    <string>Alinha a direita</string>
   </property>
  </action>
  <action name="actionCentro">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="icon">
    <iconset resource="notepad.qrc">
     <normaloff>:/images/center_align.png</normaloff>:/images/center_align.png</iconset>
   </property>
   <property name="text">
    <string>Centro</string>
   </property>
   <property name="toolTip">
    <string>Centralizar</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="notepad.qrc"/>
 </resources>
 <connections/>
</ui>
